tools/etl/tg-jdbc-driver/tg-jdbc-driver/src/main/java/com/tigergraph/jdbc/restpp/driver/QueryParser.java

```diff
@@ -507,7 +507,7 @@ public QueryParser(RestppConnection con, String query, Map<Integer, Object> para
           sb.append(getObjectStr(this.paramArray[0]));
           sb.append("\"");
         } else {
-          sb.append(tokens[4]);
+          sb.append(tokens[3]);
```

Comment:
Is this a bug fix?

Reply:
Yes, it was detected by a new unittest, a token mismatch for statement like "builtins stat_vertex_number(type=Linkto)"